src/components/Button/Button.module.scss

```diff
@@ -30,59 +32,83 @@
 }
 
 .filled_btn {
-  display: flex;
-  padding: 0.25rem 0.93rem;
-  justify-content: center;
-  align-items: center;
+  @extend .d-flex;
+  @extend .justify-content-center;
+  @extend .align-items-center;
+  padding: 0.25rem 0.9375rem;
   gap: 0.5rem;
-  border-radius: 0.37rem;
-  background: rgb(24, 154, 223);
-  box-shadow: 0rem 0.12rem 0rem 0rem rgba(0, 0, 0, 0.04);
-  border-color: red;
+  border-radius: 0.375rem;
+  background: $light-blue;
+  box-shadow: 0rem 0.125rem 0rem 0rem $shadow;
+  border-color: $black;
 
   &:hover {
-    background: rgb(179, 39, 39) !important;
+    background: $white !important;
 
     .btn_text {
-      color: blue;
+      color: $blue;
     }
   }
 
   .btn_text {
-    color: black;
+    color: $black;
     text-align: center;
-    font-family: Inter;
-    font-size: 0.87rem;
+    font-size: 0.875rem;
     font-style: normal;
     font-weight: 400;
-    line-height: 1.37rem;
+    line-height: 1.375rem;
   }
 }
 
 .white_btn {
-  display: flex;
-  padding: 0.4rem 0.938rem;
-  justify-content: center;
-  align-items: center;
-  gap: 0.625rem;
-  background-color: white;
-  border-color: var(--border);
+  @extend .d-flex;
+  @extend .justify-content-center;
+  @extend .align-items-center;
+  padding: 0.4rem 0.9375rem;
+  background-color: $white;
+  border-color: $border;
 
   &:hover {
-    border-color: blue;
+    border-color: $blue;
```

Comment:
are you sure that there is a button with this blue color ? colors should be same as figma design including colors